images/falcosidekick/tests/main.tf

```diff
@@ -0,0 +1,37 @@
+terraform {
+  required_providers {
+    oci = { source = "chainguard-dev/oci" }
+  }
+}
+
+variable "digest" {
+  description = "The image digest to run tests over."
+}
+
+data "oci_string" "ref" { input = var.digest }
+
+resource "random_pet" "suffix" {}
+
+resource "helm_release" "helm" {
+  name             = "falcosidekick-${random_pet.suffix.id}"
+  namespace        = "falcosidekick-${random_pet.suffix.id}"
+  repository       = "https://falcosecurity.github.io/charts/"
+  chart            = "falcosidekick"
+  create_namespace = true
+
+  values = [
+    jsonencode({
+      containerName = "falcosidekick"
+      image = {
+        registry   = data.oci_string.ref.registry
+        repository = data.oci_string.ref.repo
+        tag        = "latest"
```

Comment:
I think you need 'data.oci_string.ref.pseudo_tag' here, as that will grab the digest / tag for the image built at this stage of the build.

The latest tag and associated version tags are re-tags which happen later